packages/report-server/src/reportBuilder/functions/basic.ts

```diff
@@ -25,10 +25,18 @@ export const eq = (value1, value2): boolean => {
   return value1 === value2;
 };
 
+export const notEq = (value1, value2): boolean => {
+  return value1 !== value2;
+};
```

Comment:
As per my other comments I guess we don't necessarily need these functions, but happy to keep them in as they'll probably provide useful 😄 

Also worth considering that maybe we wanna migrate some of these functions into the expression parser package? Food for thought for the future 👍